analyses/cms-open-data-ttbar/analysis.py

```diff
@@ -251,11 +228,17 @@ def book_histos(
         .Filter("Sum(Jet_pt_mask) >= 4")
     )
 
+    # create columns for "good" jet pt and btag values as these columns are used several times
+    df = (
+        df.Define("Jet_pt_masked", "Jet_pt[Jet_pt_mask]")
+          .Define("Jet_btagCSVV2_masked", "Jet_btagCSVV2[Jet_pt_mask]")
```

Comment:
naming: "_masked" is not the most informative. maybe "Jet_pt_ptcut" (it's also one character shorter)? or "Jet_pt_highpt"?

Reply:
"Jet_high_pt"?

Reply:
aa, I've understood, okay

Reply:
that makes sense for `Jet_pt` but `Jet_btagCSVV2_high` doesn't work. since they are masked the same way I was trying to give them the same suffix. but we can also use two different suffixes (e.g. `_high` for one and `_highpt` for the other)

Reply:
Thanks, I already added `ptcut` for each masked column